datasets/hate_offensive/hate_offensive.py

```diff
@@ -0,0 +1,93 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Automated Hate Speech Detection and the Problem of Offensive Language."""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import pandas as pd
+
+import datasets
+
+
+_CITATION = """
+@article{article,
+author = {Davidson, Thomas and Warmsley, Dana and Macy, Michael and Weber, Ingmar},
+year = {2017},
+month = {03},
+pages = {},
+title = {Automated Hate Speech Detection and the Problem of Offensive Language}
+}
+"""
+
+_DESCRIPTION = "Technologies for abusive language detection are being developed and applied with little consideration of their potential biases. We examine racial bias in five different sets of Twitter data annotated for hate speech and abusive language. We train classifiers on these datasets and compare the predictions of these classifiers on tweets written in African-American English with those written in Standard American English. The results show evidence of systematic racial bias in all datasets, as classifiers trained on them tend to predict that tweets written in African-American English are abusive at substantially higher rates. If these abusive language detection systems are used in the field they will therefore have a disproportionate negative impact on African-American social media users. Consequently, these systems may discriminate against the groups who are often the targets of the abuse we are trying to detect."
```

Comment:
This field is intended to describe the dataset, not the article's abstract
A short, descriptive sentence stating what the dataset is about would be welcome, for example:
`This dataset contains annotated tweets for automated ate-speech recognition`